rotkehlchen/exchanges/poloniex.py

```diff
@@ -309,20 +309,24 @@ def return_trade_history(
     ) -> list[dict[str, Any]]:
         """Returns poloniex trade history"""
         limit = 100
+        max_interval_ms = DAY_IN_SECONDS * 180 * 1000
```

Comment:
this is a constant. Also the one abovr is a constant. Do not recalculate them every time the function gets executed. Move them to module level with `Final` typing.